src/main/java/io/deephaven/csv/CsvSpecs.java

```diff
@@ -126,7 +126,7 @@ public interface Builder {
         Builder putNullValueLiteralsForIndex(int index, List<String> nullValueLiteral);
 
         /**
-         * The parser to uses when all values in the column are null. Defaults to {@code Parsers#STRING}.
+         * The parser to uses when all values in the column are null. The default is {@code Parsers#STRING}.
```

Comment:
Noticed you have a few code tags where the comment is formatted as if it were a link (I'd argue "." is more appropriate than "#" for code). Link seems the better tag here.
```suggestion
         * The parser to uses when all values in the column are null. The default is {@link Parsers#STRING}.
```

Reply:
Done